manifests/init.pp

```diff
@@ -290,5 +294,74 @@
         require => Package[$tools_nox_package_name_real],
       }
     }
+
+
+    if is_bool($disable_tools_version) == true {
+      $_disable_tools_version = $disable_tools_version
+    } else {
+      $_disable_tools_version = str2bool($disable_tools_version)
+    }
+    if $_disable_tools_version == true {
+      $_disable_tools_version_string = 'true'
+    } else {
+      $_disable_tools_version_string = 'false'
+    }
+
+    if $enable_sync_driver == 'auto' {
+
+      if $working_kernel_release == 'USE_DEFAULTS' {
+        case $::operatingsystem {
+          'RedHat', 'CentOS': {
+            $_working_kernel_release = '2.6.32-358'
+          }
+          default: {
+            $_working_kernel_release = '2.6.35-22'
+          }
+        }
+      } else {
+        $_working_kernel_release = $working_kernel_release
+      }
+
+      $_kernel_release_arr = split($::kernelrelease, '[-.]')
+      $_kernel_int = $_kernel_release_arr[0] * 1000000000 + $_kernel_release_arr[1] * 1000000 + $_kernel_release_arr[2] * 1000 + $_kernel_release_arr[3]
```

Comment:
I'm not fond of this, we have several kernels that have text instead of numbers in the fourth position. Have you tested that this does not create problems?